using-mobile-extensions/mobile-core/mobile-core-api-reference.md

```diff
@@ -258,6 +258,46 @@ ACPCore.collectLaunchInfo(userInfo)
 {% endtab %}
 {% endtabs %}
 
+
+## Set Icons for local notification
+
+Set the small and large icons that will be used for notifications that are created by the SDK. The small icon appears in the status bar and is the secondary image that is displayed shown when the user sees the complete notification in the notification center. The large icon will be the primary image that is displayed when the user sees the complete notification in the notification center.
```

Comment:
- Please remove "shown" from the second sentence.
- In the 3rd sentence, please replace "will be" with "is"